src/config.rs

```diff
@@ -256,6 +259,15 @@ impl Server {
     pub fn full_url(&self) -> String {
         format!("{}:{}", self.host, self.port)
     }
+
+    pub fn add_binding_addr(&mut self, addr: &str) {
+        self.binding = Some(addr.to_string());
+    }
+
+    #[must_use]
+    pub fn get_binding(&self) -> String {
+        self.binding.clone().unwrap_or_else(|| "[::]".to_string())
```

Comment:
`[::]` is configured here and also in the `src/cli.ts.` take the `[::]` value from the cost (see previous comment)

Reply:
We can use  #[serde(default = "default_binding")].
 In the `Server` struct, you can replace the `pub binding` with this:
```rust
    #[serde(default = "default_binding")]
    resource: String,
```

And replace this function with this:
```rust
fn default_resource() -> String {
    "[::]".to_string()
}
```

This is allows us to avoid the `mut` in the `src/cli.rs` and also avoid `[::]` duplication default declaration

Reply:
@kaplanelad  `resource` you mean `binding` right?

Reply:
yea sure, like i wrote on the field struct default_binding.

thanks for correct in the example